requests/models.py

```diff
@@ -672,6 +672,7 @@ def __iter__(self):
 
     @property
     def ok(self):
+        """Returns true if :attr:`status_code` is 'OK'."""
```

Comment:
Ironically, this is the best proof that the docstring there was really necessary 😁